js/js-tip-calculator/index.js

```diff
@@ -0,0 +1,15 @@
+console.clear();
+
+// Cost of the meal
+let mealCost = 50;
+
+// Percentage of tip
+let tipPercentage = 0.1;
+
+// Tip calculation
+const tipAmount = mealCost * tipPercentage;
+
+// Total meal cost
+const totalCost = mealCost + tipAmount;
+
+console.log("// The total cost of your bill is:", totalCost, "euros");
```

Comment:
Looks like the "//" comments out the most part of the console.log. You should delete that!